modules/javafx.web/src/main/java/com/sun/webkit/FileSystem.java

```diff
@@ -52,6 +54,19 @@ private static boolean fwkFileExists(String path) {
         return new File(path).exists();
     }
 
+    private static RandomAccessFile fwkOpenFile(String path, String mode) throws FileNotFoundException {
+        return new RandomAccessFile(path, mode);
+    }
+
+    private static void fwkCloseFile(RandomAccessFile raf) throws IOException {
+        raf.close();
+    }
+
+    private static int fwkreadFromFile(RandomAccessFile raf, byte[] arr, int off, int len) throws IOException {
```

Comment:
nit: change to fwkReadFromFile

Also handle exception and return error code or error message. Also log the call and error